downstairs/src/main.rs

```diff
@@ -239,6 +255,41 @@ async fn main() -> Result<()> {
     let log = build_logger();
 
     match args {
+        Args::Clone {
+            data,
+            source,
+            trace_endpoint,
+        } => {
+            // Instrumentation is shared.
+            if let Some(endpoint) = trace_endpoint {
+                let tracer = opentelemetry_jaeger::new_agent_pipeline()
+                    .with_endpoint(endpoint) // usually port 6831
+                    .with_service_name("downstairs")
+                    .install_simple()
+                    .expect("Error initializing Jaeger exporter");
+
+                let telemetry =
+                    tracing_opentelemetry::layer().with_tracer(tracer);
+
+                tracing_subscriber::registry()
+                    .with(telemetry)
+                    .try_init()
+                    .expect("Error init tracing subscriber");
+            }
+
+            let d = build_downstairs_for_region(
+                &data,
+                false,
+                false,
+                false,
+                false,
+                true, // read_only
```

Comment:
Maybe it's time for the builder pattern... :upside_down_face:

Reply:
Made a new PR for just that:
https://github.com/oxidecomputer/crucible/pull/1152